### ISSUE TITLE
Trying to Install Gluu Server To My Refreshed Cluster Manager Throws Error

### ISSUE THREAD

**Issue by afroDC** [GluuFederation/cluster-mgr#189]
![image](https://user-images.githubusercontent.com/30447907/40501495-a5f113fa-5f4d-11e8-9087-0fbcad53690e.png)

```
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/Flask-0.12-py2.7.egg/flask/app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python2.7/dist-packages/Flask-0.12-py2.7.egg/flask/app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/usr/local/lib/python2.7/dist-packages/Flask_Login-0.4.1-py2.7.egg/flask_login/utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/clustermgr-3.1.3.post2-py2.7.egg/clustermgr/views/server.py", line 359, in install_gluu
    form.application_max_ram.data = setup_prop['application_max_ram']
KeyError: 'application_max_ram'
```

To recreate this issue have an established cluster, then remove all the nodes with the remove button. Afterwards add the same servers back. For example I had c6 as my primary and c5 and c3 as my other nodes. I removed them all and added c6 and c7 back to the cluster as fresh VM's for testing. When I hit `Install Gluu` the error is thrown.

The Cluster Manager installed is the latest Github master branch build.

**Comment by afroDC** [comment]
https://github.com/GluuFederation/cluster-mgr/blob/9ed404d5b75750aaa0b9dc2f8e40b7510d11c78a/clustermgr/forms.py#L140 It seems to be missing form data. I'm never presented with a new form to fill out, like you are when you install a new instance of Gluu

**Comment by mbaser** [comment]
closed with commit 4fa48275d9eb49f51d907b576bdeeb72827aff62
